docker-compose.build.yaml

```diff
@@ -50,3 +50,10 @@ services:
       context: airbyte-migration
       labels:
         io.airbyte.git-revision: ${GIT_REVISION}
+  temporal:
```

Comment:
Looks like you may need to add this to the `-m1` compose build file as well? https://github.com/airbytehq/airbyte/blob/master/docker-compose.build-m1.yaml

Reply:
This file should be unused. I don't think it is needed for the push operation, it was just for the up which is now unused.